### DISCUSSION TITLE
Modeling saturating partitioning into BloodCells

### DISCUSSION THREAD

**Discussion by StephanSchaller** [Open-Systems-Pharmacology/Forum#766] [Questions & Problems] [answered]
For Cyclosporin A, partitioning into blood cells saturates for high exposure (Fahr 1993):
![image](https://user-images.githubusercontent.com/25477643/86830850-79d74580-c096-11ea-86bc-0b67791eb1d5.png)

Is there any experience in how this could be implemented in MoBi as partitioning coefficients are not directly available in Building Blocks to edit their formulas.
Any workarounds / other solutions?

Best,
Stephan

**Comment by prvmalik** [comment, marked as answer]
Interesting phenomenon. Thanks for sharing Stephan. 

I did a brief read on cyclosporine, and it seems saturable intracellular binding occurs in the whole body (and blood cells). Organ-specific data is available from Tanaka, Kawai and Rowland, 1999. This data could be used to optimize the relative expression of a "protein binding partner" in each organ and the Kd globally. You could add an observer for the fudged B/P ratio and include the data from Fahr in the optimization using steady-state simulations. The partition coefficients calculated by the existing algorithms (R-R, PK-Sim Standard) are probably going to be wrong because of how much the binding contributes to distribution. What do you think?

Paul

**Comment by StephanSchaller** [comment]
Thanks for your thoughts, Paul. 
Yes, there also is an actual intracellular binding partner called Cyclophilin and the drug is highly lipoprotein bound (which might also affect partitioning). But Cyclophilin seems not to be the (only) reason for the cellular saturation.
The work from Tanaka is great to learn more on the multi-factorial nonlinear behaviour but uses data in rats, only, so far. We would need to establish the nonlinearity behaviour in humans.
But manually setting the calculated partitioning to a lower value and introducing another unspecific cellular binding partner (and an observer for the fudged cellular concentration) to capture the nonlinearity in partitioning could indeed be a workaround worth trying.

Thanks, I'll keep you updated.
